External/Plugins/HaXeContext/Context.cs

```diff
@@ -577,6 +580,21 @@ public override void RemoveClassCompilerCache()
         }
         #endregion
 
+        #region SDK
+        private InstalledSDK GetCurrentSDK()
+        {
+            return hxsettings.InstalledSDKs.FirstOrDefault(sdk => sdk.Path == currentSDK);
```

Comment:
I think that's the problem - I don't like IntelliJ's "Christmas tree" style where you have to spend your time toying with the warnings settings. Also it felt an absolute CPU hog.